docs/microbit_micropython_api.rst

```diff
@@ -157,60 +208,64 @@ Images API::
     # returns a new image created by multiplying the brightness of each pixel by n
     image * n
 
-    # built-in images.
-    Image.HEART
-    Image.HEART_SMALL
-    Image.HAPPY
-    Image.SMILE
-    Image.SAD
-    Image.CONFUSED
-    Image.ANGRY
-    Image.ASLEEP
-    Image.SURPRISED
-    Image.SILLY
-    Image.FABULOUS
-    Image.MEH
-    Image.YES
-    Image.NO
-    Image.CLOCK12 # clock at 12 o' clock
-    Image.CLOCK11
-    ... # many clocks (Image.CLOCKn)
-    Image.CLOCK1 # clock at 1 o'clock
-    Image.ARROW_N
-    ... # arrows pointing N, NE, E, SE, S, SW, W, NW (microbit.Image.ARROW_direction)
-    Image.ARROW_NW
-    Image.TRIANGLE
-    Image.TRIANGLE_LEFT
-    Image.CHESSBOARD
-    Image.DIAMOND
-    Image.DIAMOND_SMALL
-    Image.SQUARE
-    Image.SQUARE_SMALL
-    Image.RABBIT
-    Image.COW
-    Image.MUSIC_CROTCHET
-    Image.MUSIC_QUAVER
-    Image.MUSIC_QUAVERS
-    Image.PITCHFORK
-    Image.XMAS
-    Image.PACMAN
-    Image.TARGET
-    Image.TSHIRT
-    Image.ROLLERSKATE
-    Image.DUCK
-    Image.HOUSE
-    Image.TORTOISE
-    Image.BUTTERFLY
-    Image.STICKFIGURE
-    Image.GHOST
-    Image.SWORD
-    Image.GIRAFFE
-    Image.SKULL
-    Image.UMBRELLA
-    Image.SNAKE
-    # built-in lists - useful for animations, e.g. display.show(Image.ALL_CLOCKS)
-    Image.ALL_CLOCKS
-    Image.ALL_ARROWS
+**Built-in images**
+
+``Image.HEART``
+``Image.HEART_SMALL``
+``Image.HAPPY``
+``Image.SMILE``
+``Image.SAD``
+``Image.CONFUSED``
+``Image.ANGRY``
+``Image.ASLEEP``
+``Image.SURPRISED``
+``Image.SILLY``
+``Image.FABULOUS``
+``Image.MEH``
+``Image.YES``
+``Image.NO``
+``Image.CLOCK12`` clock at 12 o' clock
+...
+``Image.CLOCK1``  clock at 1 o'clock
+``Image.ARROW_N`` arrows pointing N, NE, E, SE, S, SW, W, NW
+(microbit.Image.ARROW_direction)
+...
```

Comment:
All these will be rendered in a single line, so this will look a bit weird:
![image](https://user-images.githubusercontent.com/29712657/100438350-ebda1680-3099-11eb-94e8-b2dc173af1a0.png)

Reply:
From the readthedocs build: 
![image](https://user-images.githubusercontent.com/29712657/100443157-124f8000-30a1-11eb-91c5-84565a6aa4d7.png)

Reply:
Do you prefer it as it was, taking up more room or in a list like this, but maybe with all the values and without the text?

Reply:
Or comma separated as with gestures

![image](https://user-images.githubusercontent.com/31242877/100585138-fcca9800-32e4-11eb-9865-2b85f6f53a35.png)

Reply:
Comma separated or not, pick what you think it's more readable.

My main comment was about comments like "arrows pointing N, NE, E, SE, S, SW, W, NW" in the middle of the list.

We can probably separate that in a new list as well now that it doesn't take that much space:


```
HEART, HEART_SMALL, ....

Clock:

CLOCK1, CLOCK2, ...

Arrows:

ARROW_N, ARROW_NE, ...

```

Reply:
Done